packages/es-client/tests/test-granule-upsert.js

```diff
@@ -30,7 +30,7 @@ test.after.always(async (t) => {
   await cleanupTestIndex(t.context);
 });
 
-test('upsertGranule removes deleted granule record', async (t) => {
+test.serial('upsertGranule removes deleted granule record', async (t) => {
```

Comment:
Was this causing problems?

Reply:
I don't believe so, just reverted a test.only to test.serial by mistake 😓 , fixed: 4fe4a4f63f8

Reply:
There are a couple others that seem to have been marked as `serial`. Obviously not a showstopper but e.g. https://github.com/nasa/cumulus/pull/3627/files/ceaaac0507d4ca7782e4b1ee1a0f8ff6898e9f61#diff-607630765579e07a52348c26c43dc8d7f1db891878e16b76e211d7fc04510d16R1773